src/Sidebar.js

```diff
@@ -1,46 +1,46 @@
 /**
-*
-* Sidebar
-*
-*/
-import React, { Component, createContext } from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { LIGHT_GRAY, DARK_GRAY } from './utils/colors';
-import { Link } from 'react-router-dom';
+ *
+ * Sidebar
+ *
+ */
+import React, { Component, createContext } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { LIGHT_GRAY, DARK_GRAY } from "./utils/colors";
+import { Link } from "react-router-dom";
 
 // Styles
 const Wrapper = styled.div`
-  display:flex;
-  flex-direction:column;
-  width: ${(props) => props.width}px;
-  height:100%;
-  background:${LIGHT_GRAY};
+  display: flex;
+  flex-direction: column;
+  width: ${props => props.width}px;
+  height: 100%;
+  background: ${LIGHT_GRAY};
   overflow: auto;
 `;
 
 const ItemWrapper = styled(Link)`
-  height:40px;
-  width:100%;
+  height: 40px;
+  width: 100%;
   padding-left: 20px;
   padding-right: 20px;
   text-decoration: none;
   color: black;
-  ${(props) => props.selected && `background-color: ${DARK_GRAY};`}
+  ${props => props.selected && `background-color: ${DARK_GRAY};`};
```

Comment:
The semicolon at the end is not needed. Also, the parentheses around `props` are a good habit and should be kept